pkg/service/sos/objectstorage.go

```diff
@@ -79,17 +81,24 @@ func (o *ObjectStorage) getBucketUsage(ctx context.Context) error {
 	return nil
 }
 
-func (o *ObjectStorage) fetchManagedBuckets(ctx context.Context) error {
+func (o *ObjectStorage) fetchManagedBucketsAndNamespaces(ctx context.Context) error {
 	log := ctrl.LoggerFrom(ctx)
-	log.Info("Fetching buckets from cluster")
+	log.Info("Fetching buckets and namespaces from cluster")
 
 	buckets := exoscalev1.BucketList{}
 	log.V(1).Info("Listing buckets from cluster")
-	err := o.k8sClient.List(ctx, &buckets)
+	err := o.k8sClient.List(ctx, &buckets, client.HasLabels{service.NamespaceLabel})
```

Comment:
Nitpick: Again I don't think you actually need to filter this but it's also fine if you do